transfer/transfer-06-consumer-pull-http/http-pull-consumer/consumer-configuration.properties

```diff
@@ -1,5 +1,5 @@
 edc.participant.id=consumer
-edc.ids.id=urn:connector:consumer
+edc.dsp.id=urn:connector:consumer
```

Comment:
this property does not exist anymore, it can be removed from all the configuration files, `edc.participant.id` will be enough